src/gsi/read_radar.f90

```diff
@@ -2767,3 +2767,608 @@ subroutine read_radar_l2rw_novadqc(ndata,nodata,lunout,obstype,sis,nobs)
 
 end subroutine read_radar_l2rw_novadqc
 
+!!!!!!!!!!!!!!! Xu added for l2rw thinning !!!!!!!!!!!!!!!
```

Comment:
Again, this should probably not be in this PR

Reply:
In another PR.